node/service/src/lib.rs

```diff
@@ -118,6 +118,26 @@ impl sc_executor::NativeExecutionDispatch for MoonbaseExecutor {
 	}
 }
 
+/// Trivial enum representing runtime variant
+#[derive(Clone)]
+pub enum RuntimeVariant {
+	Moonbeam,
```

Comment:
You must add #[cfg(feature = "runtime-native")] on top of each variant (except for Unrecognized)

Reply:
Should we have the ci make sure that `cargo moonbase` builds successfully? Initially I thought that would take too long. But if we do it after our copmlete build, most of the stuff should already be compiled.

Reply:
I believe changing `--features` (which `cargo moonbase` does) means we will have to build from scratch...

Reply:
fixed in 392f0ee